packages/atomic/src/components/commerce/product-template-components/atomic-product-number/atomic-product-number.tsx

```diff
@@ -0,0 +1,90 @@
+import {Product, ProductTemplatesHelpers} from '@coveo/headless/commerce';
+import {Component, Prop, Element, State, Listen} from '@stencil/core';
+import {Bindings} from '../../../../components';
+import {InitializeBindings} from '../../../../utils/initialization-utils';
+import {
+  defaultNumberFormatter,
+  NumberFormatter,
+} from '../../../common/formats/format-common';
+import {ProductContext} from '../product-template-decorators';
+
+/**
+ * The `atomic-product-number` component renders the value of a number product field.
```

Comment:
I'm not a fan of this name

suggestion: `atomic-product-numeric-field-value`

Reply:
I'm ok with atomic-product-numeric-field-value 👍